Dockerfile

```
@@ -0,0 +1,15 @@
+FROM golang:1.24-alpine AS builder
+
+ARG VERSION
+
+COPY . /build
+WORKDIR /build
+
+RUN go mod download && \
+    CGO_ENABLED=0 GO111MODULE=on go build -ldflags="-X 'main.appName=NetAssert' -X 'main.version=${VERSION}'" -v -o /netassertv2 cmd/netassert/cli/*.go && \
```

```suggestion
    CGO_ENABLED=0 GO111MODULE=on go build -ldflags="-X 'main.appName=NetAssert' -X 'main.version=${VERSION}'" -v -o /netassert ./... && \
```

`./...` lets go pick the files better

Or if there are multiple mains `./cmd/netassert/cli/...`